.github/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+# Specify location of manifest files for each package manager
```

Comment:
All the configuration options can also be added at https://app.dependabot.com/accounts/ampproject

Then there's no need for this configuration file.

We can start configuration by turning on Dependabot at https://app.dependabot.com/accounts/ampproject/add

@westonruter WDYT?

Reply:
Yeah, I suppose so. If you think that is preferable, I'm for it.

Reply:
I don't see any cons with that approach. If that's the case, I'll just update the PR to remove the Renovate config only.

Reply:
It works fine for the stories repo, and is one less config file to deal with :-)

And from what I gathered, once they integrate Dependabot into GitHub it‘ll probably be UI-only anyway.

So let‘s do it :-)